subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -156,25 +213,59 @@ static int add_static_network_config(void)
 	LOG_DBG("Adding statically configured WiFi network [%s] to internal list.",
 		creds.header.ssid);
 
-	return add_network_from_credentials_struct_personal(&creds);
+	return add_network_from_credentials_struct_personal(&creds, iface);
 #else
 	return 0;
 #endif /* defined(CONFIG_WIFI_CREDENTIALS_STATIC) */
 }
 
+static void wifi_mgmt_ext_connect_result(struct net_mgmt_event_callback *cb)
+{
+	const struct wifi_status *status = (const struct wifi_status *) cb->info;
```

Comment:
No space between `(const struct wifi_status *)` and  `cb->info`

Reply:
Done